src/Features/CustomerEffortScoreTracks.php

```diff
@@ -110,15 +145,28 @@ private function enqueue_ces_survey_for_new_product() {
 	 * Enqueue the CES survey trigger for an existing product.
 	 */
 	private function enqueue_ces_survey_for_edited_product() {
+		// Only add the JS to trigger the CES modal if
+		// this modal hasn't been dismissed or actioned yet for this action.
+		$shown_for_features = get_option( self::SHOWN_FOR_ACTIONS_OPTION_NAME, array() );
```

Comment:
Using an option for this would prevent showing the same CES prompt to multiple users. Is that desired?

Reply:
My understanding is that it should be shown for multiple users but once it is dismissed or completed it shouldn't be shown again, so I think using an option is ok.

Reply:
Yes it is, it is - https://woostartp2.wordpress.com/2020/10/12/customer-effort-score-design-i1/#comment-975